apps/system/style/net_error.css

```diff
@@ -96,21 +98,29 @@ form[role="dialog"][data-type="action"] > menu > button.icon-settings {
 
 #failed-logo {
   display: inline-block;
-  width: 13.5rem;
   height: 13.5rem;
+  width: 7.9rem;
```

Comment:
That's because the image is shown slightly bigger than it is (height is 135px rather than 125px), so the width is stretched too.
But actually, it makes more sense not to resize it in portrait mode.